recipe/meta.yaml

```diff
@@ -1,38 +1,48 @@
 {% set name = "kneed" %}
-{% set version = "0.7.0" %}
+{% set version = "0.8.5" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 340d67fa183a72b0afae19846dd246a6fb7204e8d54243d8b8d3d9932f887c6e
+  sha256: a4847ac4f1d04852fea278d5de7aa8bfdc3beb7fbca4a182fec0f0efee43f4b1
 
 build:
-  noarch: python
   number: 0
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  script: {{ PYTHON }} -m pip install . --no-build-isolation --no-deps -vv
 
 requirements:
   host:
     - python
     - setuptools
```

Comment:
```suggestion
```

`setuptools` and `hatchling` are two building systems, in this case, the upstream switched to `hatchling`, so you don't need anymore `setuptools`

https://github.com/arvkevi/kneed/blob/01c2aab60c3348ba2cda44b9385eaee7522f2434/pyproject.toml#L1-L3

Reply:
do you need wheel as well? usually that's explicit, for reasons I don't know

Reply:
only when there's `setuptools` as far as I know